mule-packager/src/main/java/org/mule/tools/api/packager/MuleProjectFoldersGenerator.java

```diff
@@ -29,6 +29,8 @@ public void generate(Path targetFolder) {
 
     createFolderIfNecessary(targetFolder.toAbsolutePath().toString(), MULE.value());
 
+    createFolderIfNecessary(targetFolder.toAbsolutePath().toString(), LIB.value());
```

Comment:
Can we please add a test for this class ?
We have one in 
https://github.com/mulesoft/mule-maven-plugin/blob/3.x/mule-packager/src/test/java/org/mule/tools/api/MuleProjectFoldersGeneratorTest.java

Reply:
There is one, I added a test